docs/source/opentelemetry.mdx

```diff
@@ -0,0 +1,214 @@
+  },
+  status: { code: 1 },
+  events: []
+}
+
+{
+  traceId: '0ed36c42718622cc726a661a3328aa61',
+  parentId: '36c6a3ae19563ec3',
+  name: 'middleware - <anonymous>',
+  id: '3776786d86f24124',
+  kind: 0,
+  timestamp: 1624650903934147,
+  duration: 63,
+  attributes: {
+    'http.route': '/',
+    'express.name': '<anonymous>',
+    'express.type': 'middleware'
+  },
+  status: { code: 0 },
+  events: []
+}
+```
+
+</ExpansionPanel>
+
+Nice! Next, we can modify this code to begin pushing trace data to an external service, such as Zipkin or Jaeger.
+
+### 3. Push trace data to a tracing system
+
+Next, let's modify the code in the [previous step](#2-configure-instrumentation) to instead push traces to a locally running instance of [Zipkin](https://zipkin.io/).
+
+> To run Zipkin locally, [see the quickstart](https://zipkin.io/pages/quickstart.html). If you want to use a different tracing system, consult the documentation for that system.
+
+First, we need to replace our `ConsoleSpanExporter` (which prints traces to the terminal) with a `ZipkinExporter`, which specifically pushes trace data to a running Zipkin instance.
+
+Install the following additional library:
+
+```bash
+npm install @opentelemetry/exporter-zipkin@0.22
+```
+
+Then, import the `ZipkinExporter` in your dedicated OpenTelemetry file:
+
+```js:title=open-telemetry.js
+const { ZipkinExporter } = require("@opentelemetry/exporter-zipkin");
+```
+
+Now we can replace our `ConsoleSpanExporter` with a `ZipkinExporter`. Replace lines 27-21 of the code in [the previous step](#2-configure-instrumentation) with the following:
+
+```js
+// Configure an exporter that pushes all traces to Zipkin
+// (This assumes Zipkin is running on localhost at the 
+// default port of 9411)
+const zipkinExporter = new ZipkinExporter({
+  // url: set_this_if_not_running_zipkin_locally
+});
+provider.addSpanProcessor(
+  new SimpleSpanProcessor(zipkinExporter)
+);
+```
+
+Now, open Zipkin in your browser at `http://localhost:9411`. You should now be able to query recent trace data in the UI!
+
+You can show the details of any operation and see a breakdown of its processing timeline by span.
+
+### 4. Update for production readiness
+
+Our example telemetry configuration assumes that Zipkin is running locally, and that we want to process every span individually as it's emitted.
+
+To prepare for production, we probably at least want to conditionally set the `url` option for the `ZipkinExporter` and replace our `SimpleSpanProcessor` with a  `BatchSpanProcessor`.
+
+You can learn more about the settings available to these and other OpenTelemetry objects in the [getting started guide](https://github.com/open-telemetry/opentelemetry-js/tree/main/getting-started).
+
+## Runnable example
+
+For a complete example of a federated graph that enables OpenTelemetry for the gateway _and_ each subgraph, [see this GitHub repo](https://github.com/apollographql/apollo-server-opentelemetry).
+
+## GraphQL-specific spans
```

Comment:
These won't be emitted once `new GraphQLInstrumentation()` has been removed, but users should see them if they have instrumented ApolloServer instances.